### DISCUSSION TITLE
the Tank can cast skills, help me

### DISCUSSION THREAD

**Discussion by kingoni1990** [Phobos-developers/Phobos#631] [Ideas and feedback]
I need make the TANK attack primary weapon, but can cast skills on the ground by "D hotkey", with cooldown time same as RPG game
I want to have Cooldown time to press deploy and deploy on target enemy
Please help me, thanks!

**Comment by zenoniations** [comment]
Superweapons should work but no Hotkeys for you.

**Reply by kingoni1990** [reply]
Superweapons only one unit, many Superweapons are annoyed, i want make a TANK cast skill same as archer maiden red alert 3, with a Deploy

**Reply by kingoni1990** [reply]
can I make a TANK, while players select this TANK----->supper weapon will appear (EX: Ion cannon), 
but while players dont select this TANK---->supper weapon will disappear (same as F skill in RA 3), still hide cooldown
while players select many TANKS----> only the first TANK--->show supper weapon
while player cast supper weapon (EX: Ion cannon), must add cast range from position of this TANK
I think that is all

**Comment by zenoniations** [comment]
can I make a TANK, while players select this TANK----->supper weapon will appear (EX: Ion cannon),
^SW.AuxUnit= feature is coming, it should help.

but while players dont select this TANK---->supper weapon will disappear (same as F skill in RA 3), still hide cooldown
^Sounds impossible

while players select many TANKS----> only the first TANK--->show supper weapon
^Same as above.

while player cast supper weapon (EX: Ion cannon), must add cast range from position of this TANK
^SW.RangeMaximum= for SW range
SW.SW.Designators= for the tank to provide SW range.
